doc/GETTING_STARTED.md

```diff
@@ -0,0 +1,482 @@
+
+#### Transports
+
+Libp2p uses Transports to establish connections between peers over the network. You can configure 1 Transport, or as many as you like. Supporting more Transports will improve the ability for other nodes on the network to communicate with you.
+
+You should select Transports according to the runtime where your application will run. You can see a list of some of the available Transports in the [configuration readme](./CONFIGURATION.md#transport). We are going to install `libp2p-websockets`, as it can be used in both Node.js and the browser.
+
+Start by installing `libp2p-websockets`:
+
+```sh
+npm install libp2p-websockets
+```
+
+Now that we have the module installed, let's configure libp2p to use the Transport. We'll use the [`Libp2p.create`](./API.md#create) method, which takes a single configuration object as its only parameter. We can add the Transport by passing it into the `modules.transport` array:
+
+```js
+const Libp2p = require('libp2p')
+const WebSockets = require('libp2p-websockets')
+
+const node = await Libp2p.create({
+  modules: {
+    transport: [WebSockets]
+  }
+})
+```
+
+As new Transports are created, you may wish to reevaluate your needs and select the latest Transports that best suit your requirements. You may wish to remove the Transports you had before, or simply append the new Transports to `modules.transport` in order to establish connections with peers that support either.
+
+<details><summary>Read More</summary>
+If you want to know more about libp2p transports, you should read the following content:
+
+- https://docs.libp2p.io/concepts/transport
+- https://github.com/libp2p/specs/tree/master/connections
+</details>
+
+#### Connection Encryption
+
+Encryption is an important part of communicating on the libp2p network. Every connection must be encrypted to help ensure security for everyone. As such, Connection Encryption (Crypto) is a required component of libp2p.
+
+There are a growing number of Crypto modules being developed for libp2p. As those are released they will be tracked in the [available Connection Encryption](./CONFIGURATION.md#connection-encryption) section of the configuration readme. For now, we are going to configure our node to use the `libp2p-secio` module.
+
+```sh
+npm install libp2p-secio
+```
+
+With `libp2p-secio` installed, we can add it to our existing configuration by importing it and adding it to the `modules.connEncryption` array:
+
+```js
+const Libp2p = require('libp2p')
+const WebSockets = require('libp2p-websockets')
+const SECIO = require('libp2p-secio')
+
+const node = await Libp2p.create({
+  modules: {
+    transport: [WebSockets],
+    connEncryption: [SECIO]
+  }
+})
+```
+
+<details><summary>Read More</summary>
+If you want to know more about libp2p connection encryption, you should read the following content:
+
+- https://docs.libp2p.io/concepts/secure-comms
+- https://github.com/libp2p/specs/tree/master/connections
+</details>
+
+#### Multiplexing
```

Comment:
I think we need to split multiplexing configuration and protocol creation into separate sections.

This Multiplexing section also feels like it's diving a little too deep into what multiplexing is. I think we should reduce the up front language and the leverage the `Read More` section to  send folks to more detailed information.

Reply:
I think the protocol creation section should also come after starting and stopping your node. That way if people have issues with their early configuration, it can be caught prior to building out a custom protocol.

Do you think it would be more helpful to point people to the protocol example, instead of building that out here? I think `Getting Started` can focus on creating a first node, and connecting it to the public network with basic peer discovery and new connection logging. Then we could point to the examples folder. This would also put us in a good spot to revamp those in the future, where all examples are based off of the configuration from the getting started guide.